integration-cli/docker_cli_daemon_test.go

```diff
@@ -1804,3 +1804,44 @@ func (s *DockerDaemonSuite) TestDaemonStartWithDefalutTlsHost(c *check.C) {
 		c.Fatalf("docker version should return information of server side")
 	}
 }
+
+// Test Daemon starts when the repositories-aufs file is removed
+func (s *DockerDaemonSuite) TestDaemonStartWithNoRepoConf(c *check.C) {
```

Comment:
@cpuguy83 @LK4D4 seems like adding these dependencies to both the tests still fails the tests only on experimental jenkins. I am not familar with diff's between experimental and janky build. Anything else you would recommend looking into as the tests pass on janky build ? Thanks.